samples/ClientSample/HubSample.cs

```diff
@@ -79,19 +87,15 @@ internal static void Register(CommandLineApplication app)
             return 0;
         }
 
-        private static async Task<HubConnection> ConnectAsync(string baseUrl)
+        private static async Task ConnectAsync(HubConnection connection)
         {
             // Keep trying to until we can start
             while (true)
             {
-                var connection = new HubConnectionBuilder()
-                                .WithUrl(baseUrl)
-                                .WithConsoleLogger(LogLevel.Trace)
-                                .Build();
+
                 try
                 {
                     await connection.StartAsync();
-                    return connection;
                 }
```

Comment:
Don't you need to return here? Otherwise this method will never return... or am I missing something?

Reply:
The return value was changed from `Task<HubConnection>` to `Task`

Reply:
What @fretje means is that this is an infinite loop:

```c#
while (true)
{
    try
    {
        await connection.StartAsync();
    }
    catch (Exception)
    {
    }
}
```

Reply:
@acjh Thanks! Yes you guys are right, this is broken. Would either of you like to make a PR to fix it?